G/GDAL/build_tarballs.jl

```diff
@@ -132,20 +135,20 @@ hdf5_platforms = expand_cxxstring_abis(hdf5_platforms)
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("GEOS_jll"; compat="~3.11"),
-    Dependency("PROJ_jll"; compat="~900.100"),
+    Dependency("GEOS_jll"; compat="3.11.2"),
+    Dependency("PROJ_jll"; compat="900.100.0"),
     Dependency("Zlib_jll"),
     Dependency("SQLite_jll"),
     Dependency("LibPQ_jll"),
     Dependency("OpenJpeg_jll"),
     Dependency("Expat_jll"; compat="2.2.10"),
     Dependency("Zstd_jll"),
-    Dependency("Libtiff_jll"; compat="4.3"),
+    Dependency("Libtiff_jll"; compat="4.4.0"),
```

Comment:
```suggestion
    Dependency("Libtiff_jll"; compat="4.5.1"),
```

Bumping compat to 4.5.1, since that was breaking on Windows, see:
- https://github.com/JuliaRegistries/General/pull/86099
- https://github.com/JuliaPackaging/Yggdrasil/pull/6929